styles/components/LoggedInAs.module.less

```diff
@@ -75,13 +76,17 @@
   composes: icon-logout from '../core/icons.module.less';
 }
 
+.updateIcon {
+  composes: icon-refresh from '../core/icons.module.less';
+}
+
 .dropdown {
   border: 1px solid @gray-light;
   cursor: default;
   position: absolute;
   background: white;
-  top: 45px;
-  left: 431px;
+  top: 42px;
+  left: ~"calc(100% - 164px)";
```

Comment:
`~` tells LESS not to process the next value. Since we're bundling Chromium, it's safe to use `calc()`. (actually, `calc()` has pretty solid support across browsers now anyway)